include/moqt/client.h

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright (C) 2024
+ *  Cisco Systems, Inc.
+ *  All Rights Reserved
+ */
+
+#pragma once
+
+#include <moqt/config.h>
+#include <moqt/core/transport.h>
+
+namespace moq::transport {
+    using namespace qtransport;
+
+    /**
+     * @brief MoQT Client
+     *
+     * @details MoQT Client is the handler of the MoQT QUIC transport IP connection.
+     */
+    class Client : public Transport
+    {
```

Comment:
my preference would be to add an api to set the transport instead .. This would enable dependency injection flows and enables easy testing with fakes for things